Source/Application/AnlApplicationMainMenuModel.cpp

```diff
@@ -222,8 +262,27 @@ void Application::MainMenuModel::updateAppleMenuItems()
     extraAppleMenuItems.addCommandItem(&commandManager, CommandIDs::helpOpenPluginSettings);
     extraAppleMenuItems.addCommandItem(&commandManager, CommandIDs::helpOpenKeyMappings);
     addGlobalSettingsMenu(extraAppleMenuItems);
+    // This is a hack to update the main menu when the translation changes
+    juce::MenuBarModel::setMacMainMenu(nullptr, nullptr);
     juce::MenuBarModel::setMacMainMenu(this, &extraAppleMenuItems);
 }
 #endif
 
+juce::File Application::MainMenuModel::getEmbeddedTranslationsDirectory()
+{
+    auto const exeFile = juce::File::getSpecialLocation(juce::File::SpecialLocationType::currentExecutableFile);
+#if JUCE_MAC
+    return exeFile.getParentDirectory().getSiblingFile("Resources").getChildFile("Translations");
+#else
+    return exeFile.getSiblingFile("Translations");
+#endif
+}
+
+juce::File Application::MainMenuModel::getUserTranslationsDirectory()
+{
+    auto const directory = Properties::getFile("").getSiblingFile("Partiels").getChildFile("Translations");
+    directory.createDirectory();
```

Comment:
There is no need to create the directory. If it doesn't exist, it means it is empty and we can let the user create it if necessary. The method can return at the previous line directly.